nemo_skills/prompt/config/lean4/nat-to-lean4-math.yaml

```diff
@@ -25,9 +25,12 @@ user: |-
   Answer is: {expected_answer}
   ```lean4
   import Mathlib
+  import Aesop
```

Comment:
Let's try to have a single source of truth for this header. It's currently referenced in quite a few different places and it's very easy to forget to modify one of them. Instead, can we put it inside some util function for lean and then import it in the evaluate as well as when we prepare the data? And as long as it's saved inside data files through prepare script, we can reference it here with {header}

Reply:
If this header is the same in all places, I just want to put it in some separate file (in a function or constant variable) and import it from there wherever it needs to be used. The only problem is the yaml files, for which it should instead come from the data jsonl directly (and there it should be put by prepare.py function)